tenable/io/v3/vm/files.py

```diff
@@ -0,0 +1,53 @@
+'''
+Files
+=====
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`file <file>` API endpoints.
+
+Methods available on ``tio.v3.vm.files``:
+
+.. rst-class:: hide-signature
+.. autoclass:: FileAPI
+    :members:
+'''
+from typing import BinaryIO
+
+from tenable.base.endpoint import APIEndpoint
+
+
+class FileAPI(APIEndpoint):
+
+    _path = 'api/v3/file'
+    _conv_json = True
+
+    def upload(self, fobj: BinaryIO, encrypted: bool = False) -> str:
+        '''
+        Uploads a file into Tenable.io.
+
+        :devportal:`file: upload <file-upload>`
+
+        Args:
+            fobj (FileObject):
+                The file object intended to be uploaded into Tenable.io.
+            encrypted (bool, optional):
+                If the file is encrypted, set the flag to True.
+
+        Returns:
+            :obj:`str`:
+                The fileuploaded attribute
+
+        Examples:
+            >>> with open('file.txt') as fobj:
```

Comment:
I'd change this filename to match with something closer to what would be expected by the API, much like you have in the test

Reply:
Updated the change.